sdk/servicebus/azure-servicebus/api_review.py

```diff
@@ -0,0 +1,341 @@
+    def get_session_id(self) -> str:
+    def get_session_state(self) -> str:  # This can be made to property if property is more pythonic way
+    def set_session_state(self, state : str) -> None:  # This can be made to property if property is more pythonic way
+    def renew_lock(self) -> None:
+    @property
+    def expired(self) -> bool:
+
+
+class Transaction:
+    def __init__(self) -> None:
+    def __enter__(self):
+    def __exit__(self):
+    def begin(self) -> None:
+    def commit(self) -> None:
+    def abort(self) -> None:
+
+
+class Message:
+    def __init__(self, body : str, encoding : str = 'UTF-8', **kwargs) -> None:
+    def __str__(self):
+
+    # @properties
+    def settled(self) -> bool:  # read-only
+    def enqueued_time(self) -> datetime:  # read-only
+    def scheduled_enqueue_time(self) -> datetime:  # read-only
+    def sequence_number(self) -> int:  # read-only
+    def partition_id(self) -> str:  # read-only
+    def locked_until(self) -> datetime:  # read-only
+    def expired(self) -> bool:  # read-only
+    def lock_token(self) -> str:  # read-only
+    def body(self) -> Union[bytes, Generator[bytes]]:  # read-only
+    def partition_key(self, value : str):
+    def partition_key(self) -> str:
+    def via_partition_key(self, value: str):
+    def via_partition_key(self) -> str:
+    def session_id(self, value : str):
+    def session_id(self) -> str:
+    def time_to_live(self, value : Union[float, timedelta]):
+    def time_to_live(self) -> datetime.timedelta:
+    def annotations(self, value : dict[str, Any]):
+    def annotations(self) -> dict[str, Any]:
+    def user_properties(self, value : dict[str, Any]):
+    def user_properties(self) -> dict[str, Any]:
+    def enqueue_sequence_number(self, value : int):
+    def enqueue_sequence_number(self) -> int:
+
+    # Methods
+    def schedule(self, schedule_time : datetime) -> None:
+    def renew_lock(self) -> None:
+    def complete(self) -> None:
+    def dead_letter(self, description : str = None) -> None:
+    def abandon(self) -> None:
+    def defer(self) -> None:
+
+
+class BatchMessage(Message):
+# inherited from Message
+    def try_add(self, message: Message) -> None:
+
+
+class PeekMessage(Message):
+# inherited from Message, and cannot settle/lock the message
+
+
+class DeferredMessage(Message):
+    # interited from Message
+    # its own @properties
+    def settled(self) -> bool:  # read-only
+
+
+class ReceiveSettleMode:
+    PeekLock
+    ReceiveAndDelete
+
+
+class TransportType(Enum):
+    Amqp
+    AmqpOverWebsocket
+
+
+class ServiceBusConnection:
+    def __int__(
+        self,
+        fully_qualified_namespace,
+        entity_name,
+        token_credential
+    ) -> None:
+    def from_connection_string(
+        self,
+        conn_str,
+        entity_name=None
+    ) -> ServiceBusConnection:
+
+class ServiceBusSharedKeyCredential:
+    def __init__(self, policy, key):
+    def get_token(self, *scopes, **kwargs):
+
```

Comment:
Add meta class `ServiceBusClient` with mgmt apis like `create_queue`, `create_topic`...just for showing